CHANGELOG.md

```diff
@@ -15,7 +25,7 @@ All notable changes to this project will be documented in this file.
 - [spring-security] and [spring-security-starter] `HybridIdentityServicesAutoConfiguration` supports Identity service configuration alone, by setting up `IasJwtDecoder`
 
 ## 2.8.12 and 0.1.5 [BETA]
-- The following dependencies were updated:
+- The following dependencies are updated:
```

Comment:
revert it back to "were"

Reply:
I thought it fits better to the present style, we normally have in CHANGELOG. Would you say its wrong?

Reply:
for me personally "were updated" sounds better as it has been already done and is finished process. Also checking changelog "were updated" was used previously describing dependency updates.

Reply:
agreed on  subsection "Dependency Upgrades"